BrainPortal/app/views/data_providers/_one_data_provider_table.html.erb

```diff
@@ -76,15 +76,19 @@
   %>
 
   <% t.column("Alive?", :alive) do |dp| %>
-    <% if dp.online? %>
-      <%= staggered_loading "span", url_for(:action  => :is_alive, :id  => dp.id), :error_message  => "<span class=\"warning\">???</span>" do %>
-        <span class="loading_message"><%= html_colorize("(...)","magenta") %></span>
+    <%= on_click_ajax_replace( { :element  => "div",
+                                                :url      => url_for(:controller => :data_providers, :action  => :is_alive, :id => dp.id)
+                                             },
+                                             {
+                                               :before   => "<div class='loading_message'>Loading...</div>",
+                                               :class => "dp_alive"
+                                             }
+                                            ) do %>
+      <a>check</a>
```

Comment:
The browse links appear the same, with an href attribute, which we don't want here because it does not go to a separate page. I'm not sure what you mean.